### DISCUSSION TITLE
Weird log history for HM-800

### DISCUSSION THREAD

**Discussion by CommanderRedYT** [tbnobody/OpenDTU#1437] [General]
I have a HM-800 and I am wondering, do other people with this Inverter also have the "36"-events? I actually bought everything as one kit, and considering that it is winter and that the PV installation is not really placed optimal, I am wondering how this might happen.

![image](https://github.com/tbnobody/OpenDTU/assets/43087936/00407a63-4979-49d9-98b7-6c8a596802d2)
![image](https://github.com/tbnobody/OpenDTU/assets/43087936/513c9848-20a8-45f6-8fee-3180982e4714)

**Comment by stefan123t** [comment]
Problem ist bekannt und die biaher unbekannten events aus warn_code.json in https://github.com/tbnobody/OpenDTU/issues/121#issuecomment-1752124208 dokumentiert

**Reply by CommanderRedYT** [reply]
Ich weiß, aber mir ging's eher darum warum dieser Fehler/diese Warnung auftreten könnte, was sie bedeutet steht ja auch wie im Screenshot daneben. Im sinne von es scheint kaum die Sonne und trotzdem ist Überspannung oder Overcurrent

**Reply by Tiese** [reply]
Bei mir lag es am FI-Schalter!
0,03A gegen 0,3A getauscht - seitdem keine Fehlermeldung mehr :-)

**Reply by CommanderRedYT** [reply]
Meinst du das der FI geflogen ist? Ich versteh nicht ganz wie sowas einen Zusammenhang mit einem FI hat :sweat_smile:

**Reply by Tiese** [reply]
Erst kam die Fehlermeldung 36, manchmal auch 37, 46 oder auch 12 oder 13 und dann erst flog der FI.
Der war dann ganz schön warm?!
Der neue bleibt kalt, fliegt auch nicht mehr raus und Fehler sind auch keine mehr!
Wie da der Zusammenhang ist weiß ich auch nicht.
Da der Elektriker meinte es könnte auch am Wechselrichter liegen, hatte ich den auch getauscht - daran lag es aber nicht.

**Reply by iomax** [reply]
The same here (Hm-350), and it's hard to believe in going 'over' ( Voltage or Current ) nowadays.

**Reply by CommanderRedYT** [reply]
Maybe false-trigger? I mean the software on the inverter is from 2021, and im surely not buying a original DTU to update it :)
